1351_count_negative_numbers_in_a_sorted_matrix.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+
+class Solution:
+    
+    def countNegatives(self, grid: List[List[int]]) -> int:
+        yoko = len(grid[0])
+        tate = len(grid)
+
+        i = 0
+        j = yoko -1 
+        count = 0
+        while i < tate and j >= 0:
+            if grid[i][j] < 0:
+                count = count + 1
+
```

Comment:
Don't insert a new line.

Reply:
Sure, I revised my code. Thanks for teaching.